components/components/view-modes/highlighted-small/node--highlighted-small.html.twig

```diff
@@ -1,30 +1,33 @@
-<?php
+{#
 /**
  * @file
  * This file creates the 'highlighted-small' view mode for our styleguide.
  */
-?>
-
-<style>
-  <?php include $_SERVER['DOCUMENT_ROOT'] . '/base.css'; ?>
-  <?php include $_SERVER['DOCUMENT_ROOT'] . '/components/view-modes/highlighted-small/style.css'; ?>
-</style>
-
-<script>
-  <?php include 'script.js'; ?>
-</script>
+#}
 
 <article role="article" class="node node--type-recipe node--view-mode-highlighted-small" about="/pages/recipe">
   <div class="node__content">
+
     <div class="field field--name-field-image field--type-image field--label-hidden field__item">
       <a href="/pages/recipe" rel="bookmark">
-        <?php include $_SERVER['DOCUMENT_ROOT'] . '/components/image-styles/highlighted-small.php'; ?>
+        {% include '../../image-styles/highlighted-small.html.twig' %}
       </a>
     </div>
-    <h2>
-      <a href="/pages/recipe" rel="bookmark">
-        <span class="field field--name-title field--type-string field--label-hidden"><?php print $title; ?></span>
-      </a>
-    </h2>
+
+    <div class="node__meta">
+      <div class="field field--name-field-difficulty field--type-list-string field--label-inline">
```

Comment:
At this place the content type (feature, recipe) name is in the design in the highlighted small view mode. But i think it is better to keep this because the read more link also says what content type this is.

(Well it is ok for recipe but what will be her in a feature?)

Reply:
her = here

Reply:
I'm taking the styleguide pdf as my source for this.

![screen shot 2017-10-14 at 10 15 06](https://user-images.githubusercontent.com/2183332/31574470-a41235ca-b0c8-11e7-9af5-c6ad783550d1.png)

Reply:
VIEW ARTICLE >
Do we agree that this makes sense?